src/pages/Admin/Venue/EntranceInput.tsx

```diff
@@ -128,7 +130,10 @@ const EntranceInput: React.FC<EntranceInputProps> = ({
   };
 
   return (
-    <div className="input-container" style={{ marginBottom: "1.5rem" }}>
+    <div
+      className={`input-container ${className}`}
+      style={{ marginBottom: "1.5rem" }}
+    >
```

Comment:
```suggestion
    <div
      className={`EntranceInput input-container ${className}`}
      style={{ marginBottom: "1.5rem" }}
    >
```

Makes it possible to add SCSS instead of `style` prop. Example:
```
 .EntranceInput { margin-bottom: 1.5rem; }
```

Reply:
Fixed in: https://github.com/sparkletown/sparkle/pull/2012/commits/a404f57bd688146d8e490c8728fbf54689a7fafc